### PR TITLE
LLL: better error reporting in (include)

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@benjaminion @zigguratt can you please double check this?

**Comment by zigguratt** [comment]
I can certainly do that but I'll be unavailable for the next couple of days. I'll take a look after that. Thanks for your continued LLL efforts, @axic! I'm still 100% committed to the advancement of the language. In fact, I'm working on LLL-related tasks full time now. I'm currently writing a series of articles about the structure of an LLL contract and how it relates to both Solidity and assembly.

**Comment by benjaminion** [comment]
Working nicely for me.

File: liblll/CodeFragment.cpp [hunk 1]
```diff
@@ -192,7 +192,13 @@ void CodeFragment::constructOperation(sp::utree const& _t, CompilerState& _s)
 		{
 			if (_t.size() != 2)
 				error<IncorrectParameterCount>();
-			m_asm.append(CodeFragment::compile(contentsString(firstAsString()), _s).m_asm);
+			string fileName = firstAsString();
+			if (fileName.empty())
+				error<InvalidName>("Empty file name provided");
+			string contents = contentsString(fileName);
```
Comment (chriseth) [inline liblll/CodeFragment.cpp:198]: I did not know this was here. This permits reading any file from the filesystem. If lll is added to the json compiler, this security flaw is extended to solidity.

**Review by chriseth — APPROVED** [review]

**Pull request merged**